tests/host/test_ping.py

```diff
@@ -0,0 +1,70 @@
+import asyncio
+import secrets
+
+import pytest
+
+from libp2p.host.ping import ID, PING_LENGTH, handle_ping
+from libp2p.peer.peerinfo import info_from_p2p_addr
+from tests.utils import set_up_nodes_by_transport_opt
+
+
+def _add_ping_to(host):
+    host.set_stream_handler(ID, handle_ping)
+
+
+@pytest.mark.asyncio
+async def test_ping_once():
+    transport_opt_list = [["/ip4/127.0.0.1/tcp/0"], ["/ip4/127.0.0.1/tcp/0"]]
+    (host_a, host_b) = await set_up_nodes_by_transport_opt(transport_opt_list)
+
+    # NOTE: this will fail after we add ping as a default handler
+    # as a forced reminder to fix this test by removing the calls to
+    # `_add_ping_to`
+    assert host_a.get_mux().handlers == {}
+    assert host_b.get_mux().handlers == {}
```

Comment:
Probably we can just remove this check if it is not necessary?

Reply:
this was intentional just to add some extra safety to this PR... it will disappear as soon as we merge the other PR that changes this condition :)